web/core/components/workspace/sidebar/projects-list.tsx

```
@@ -175,20 +177,25 @@ export const SidebarProjectsList: FC = observer(() => {
+                  tooltipContent=""
+                  position="right"
+                  disabled={!isCollapsed}
+                >
                   <>
                     {isCollapsed ? (
                       <Briefcase className="flex-shrink-0 size-3" />
                     ) : (
-                      <span className="text-xs font-semibold">YOUR PROJECTS</span>
+                      <span className="text-xs font-semibold">{t("your_projects").toUpperCase()}</span>
                     )}
                   </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)